apps/backend/test/mocks/classes/cash_deposit_mock.ts

```diff
@@ -19,7 +19,7 @@ export class CashDepositMock extends CashDepositEntity {
     this.deposit_date = dateRange.maxDate;
     this.metadata = metadata;
     this.deposit_amt_cdn = amount;
-    this.pt_location_id = location.pt_location_id;
+    this.pt_location_id = location.banks[0].bank_id;
```

Comment:
Is this always true that pt_location_id corresponds to this? A location won't always have a set of banks right?

I know this is a mock, but I'm curious

Reply:
Yes, a location always has a set of banks.

Reply:
Hmm, you are correct though, not all banks will necessarily have an entry in the cash deposit files. I am wondering if we should include the method on this table.
(all banks have a location_id, all location_ids have banks, all cash_deposit rows have a pt_location_id which corresponds to a bank_id which corresponds to a location, but since we're storing the bank_id for non cash as well then it's possible that banks[0] would not correspond to a cash type bank_id.)